docs/buildah-from.md

```diff
@@ -443,6 +443,25 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Buildah to label the content with a private unshared label.
 Only the current container can use a private volume.
 
+The `:O` flag tells buildah to mount the directory from the host as a
```

Comment:
Ditto changes from bud here too please.

Reply:
Yes I am copying the same content between the two man pages.